### PR TITLE
Saas 12.5 fix grouped views las

### PR DESCRIPTION
**Pull request by smetl**
[FIX] account: Prevent messing invoices when writing account_id

When writing an account from the editable groupby list view, prevent
the user to messing up the invoices by writing receivable accounts everywhere.

[FIX] account: Fix editable fields on editable grouped list views

Nothing is editable on account.move.line tree view.
On grouped tree views, only account_id, name, ref, tag_ids, date_maturity are editable.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/views/account_move_views.xml [hunk 1]
```diff
@@ -138,26 +138,26 @@
...
-                    <field name="tag_ids" widget="many2many_tags" width="0.5" optional="hide"/>
+                    <field name="ref" optional="show" readonly="1"/>
+                    <field name="name" optional="show" readonly="1"/>
+                    <field name="analytic_account_id" groups="account.group_account_user" optional="show" readonly="1"/>
+                    <field name="analytic_tag_ids" widget="many2many_tags" groups="analytic.group_analytic_tags" optional="hide" readonly="1"/>
+                    <field name="tax_ids" widget="many2many_tags" width="0.5" optional="show" readonly="1"/>
+                    <field name="debit" readonly="1" sum="Total Debit"/>
+                    <field name="credit" readonly="1" sum="Total Credit"/>
+                    <field name="amount_currency" readonly="1" groups="base.group_multi_currency" optional="hide"/>
+                    <field name="tag_ids" widget="many2many_tags" width="0.5" optional="hide" readonly="1"/>
```
Comment (qdp-odoo) [inline addons/account/views/account_move_views.xml]: readonly=0 here on tag_ids

File: addons/account/views/account_move_views.xml [hunk 2]
```diff
@@ -138,26 +138,26 @@
             <field name="arch" type="xml">
                 <tree string="Journal Items" create="false">
                     <field name="date" optional="show"/>
-                    <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
                     <field name="journal_id" options='{"no_open":True}'/>
                     <field name="move_id" optional="show"/>
-                    <field name="account_id" options="{'no_open': True, 'no_create': True}" domain="[('company_id', '=', company_id)]" groups="account.group_account_user"/>
-                    <field name="partner_id" optional="show"/>
+                    <field name="account_id" readonly="1" options="{'no_open': True}" groups="account.group_account_user"/>
```
Comment (qdp-odoo) [inline addons/account/views/account_move_views.xml]: readonly=0 on account_id, as we said we don't trigger onchanges

File: addons/account/views/account_move_views.xml [hunk 3]
```diff
@@ -264,7 +250,10 @@
             <field name="mode">primary</field>
             <field name="inherit_id" ref="account.view_move_line_tree_grouped"/>
             <field name="arch" type="xml">
-                <field name="account_id" position="attributes"><attribute name="optional">hide</attribute></field>
+                <field name="account_id" position="attributes">
+                    <attribute name="optional">hide</attribute>
+                    <attribute name="readonly">1</attribute>
+                </field>
```
Comment (qdp-odoo) [inline addons/account/views/account_move_views.xml]: this is to remove as well

Comment (smetl) [reply addons/account/views/account_move_views.xml]: No because the move lines are grouped by account on the general ledger view

**Comment by smetl** [comment]
@qdp-odoo changes made

**Comment by qdp-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**